lib/Ama.pm

```diff
@@ -96,6 +102,7 @@ $self->plugin("OAuth2Accounts" => {
   $r->delete('/questions/:question_id')->to('questions#remove')->name('remove_question'); # Delete from DB and redirect to questions
   $r->delete('/removeAll')->to('questions#removeAll')->name('removeAll'); #Delete every question
```

Comment:
@jsiems I know this isn't your PR, but I just made a comment which caught my attention on this.  I think this should be under $api instead of $r, making the route /api/removeAll instead of /removeAll.  Moreover, I don't think you should have mixed case in the URL route.  Moreover, I think "All" should be a parameter passed to /api/remove.  Let me know if you'd like more discussion on this.